lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -144,20 +152,16 @@
 import logging
 
 
-def netconf_edit_config(m, xml, commit, retkwargs):
-    if ":candidate" in m.server_capabilities:
-        datastore = 'candidate'
-    else:
-        datastore = 'running'
+def netconf_edit_config(m, xml, commit, retkwargs, datastore):
```

Comment:
parameter retkwargs is not used;
should avoid using function parameters which are not implemented...